tests/test_cases.yaml

```
@@ -168,3 +168,38 @@ tests:
       exit_code: 0
+  - name: atmos vendor pull 
+    enabled: true
+    description: "Ensure atmos vendor pull command executes without errors and files are present."
+    workdir: "../examples/tests/test-vendor/"
+    command: "atmos"
+    args:
+      - "vendor"
+      - "pull"
+    expect:
+      file_exists: [
+          "./components/terraform/github/stargazers/main/main.tf",
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)